addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF

```diff
@@ -14,6 +14,7 @@ Bundle-NativeCode: lib/x86-64/win/alljoyn_java.dll;processor=amd64;osname=win32;
  lib/arm/linux/liballjoyn_java.so;processor=arm;osname=linux,*
 Import-Package: com.google.common.collect,
  org.apache.commons.io,
+ org.eclipse.jdt.annotation;resolution:=optional,
```

Comment:
Is this now required for every bundle should I refuse bindings without this? 
Should we add static analysis for this?